Src/CSharpier.MsBuild/CSharpier.MsBuild.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="../../CSharpier.Targets"/>
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
```

Comment:
You need the `netstandard2.0` here. Otherwise it can't be used to format projects that don't also target `net5.0` themselves